packages/embark-ui/src/components/Accounts.js

```diff
@@ -2,17 +2,35 @@ import React from 'react';
 import {Row, Col, Card, CardHeader, CardBody} from 'reactstrap';
 import {Link} from 'react-router-dom';
 import PropTypes from 'prop-types';
+import Pagination from './Pagination';
 
 import CardTitleIdenticon from './CardTitleIdenticon';
 
-const Accounts = ({accounts}) => (
+function displayTruncatedBalance (balance) {
+  const max_display_length = 20;
+  if (balance.toString().length > max_display_length) {
```

Comment:
Could you reverse the condition and early return here :)

Reply:
Done.